controller/source/create.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+from controller.source import bp, db
+from flask import request
+
+
+def valid(x):
+    return x
```

Comment:
Is this intended to be a placeholder for the future?

Reply:
Yes.  I'm not sure what a "valid" name/url is for a harvest source.  But we'll have to define those specifications at some point in the future.